cmd/cortex-ingester/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/prometheus/common/log"
+
+	"github.com/weaveworks/cortex"
+	"github.com/weaveworks/cortex/chunk"
+	"github.com/weaveworks/cortex/ingester"
+	"github.com/weaveworks/cortex/ring"
+	"github.com/weaveworks/cortex/server"
+	"github.com/weaveworks/cortex/util"
+)
+
+func main() {
+	var (
+		serverConfig               = server.Config{}
```

Comment:
The `var serverConfig = server.Config{}` style here and below is just a bit unusual. More usual would be either `var serverConfig server.Config` or `serverConfig := server.Config{}` (not a mix of `var` and explicit assignment of zero value).